### PR TITLE
[DNNL] 3D Fully-Connected

### PR DESCRIPTION
**Pull request by Sand3r-**
Introduces support for 3D FC in both fp32 and int8.

### CODE REVIEW

File: paddle/fluid/operators/mkldnn/fc_mkldnn_op.cc [hunk 1]
```diff
@@ -12,6 +12,7 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License. */
 
+#include <mkldnn/include/mkldnn_debug.h>
```
Comment (wojtuss) [inline paddle/fluid/operators/mkldnn/fc_mkldnn_op.cc]: This header seems to be redundant.

Comment (Sand3r-) [reply paddle/fluid/operators/mkldnn/fc_mkldnn_op.cc]: Done.

**Review by wojtuss — DISMISSED** [review]
LGTM

**Comment by Sand3r-** [comment]
@wzzju Could you please start the review?

**Review by wzzju — DISMISSED** [review]
LGTM.

**Review by wozna — APPROVED** [review]
LGTM

**Review by jczaja — APPROVED** [review]
LGTM

**Review by luotao1 — APPROVED** [review]
LGTM

**Pull request merged**